install_config/cluster_metrics.adoc

```diff
@@ -304,13 +304,15 @@ appended to the prefix starting from 1.
 |The persistent volume claim size for each of the Cassandra nodes.
 
 |`openshift_metrics_cassandra_pvc_storage_class_name`
-|If you want to explicitly set the storage class, you must not set
-`openshift_metrics_cassandra_storage_type` to `emptydir` or `dynamic`.
+|Specify the storage class to use. If you want to explicitly set the storage class, 
+use `openshift_metrics_cassandra_storage_type=pv`. This parameter does not work 
+if `openshift_metrics_cassandra_storage_type` is `emptydir` or `dynamic`.
```

Comment:
Tested on a clouldprovider enabled 311 cluster, openshift_metrics_cassandra_storage_type  is emptydir or dynamic can work with openshift_metrics_cassandra_pvc_storage_class_name
example:
***************
openshift_metrics_cassandra_storage_type=emptydir(if set to emptydir, no PV is created)
openshift_metrics_cassandra_pvc_storage_class_name=standard
***************
openshift_metrics_cassandra_storage_type=dynamic
openshift_metrics_cassandra_pvc_storage_class_name=standard
***************